.github/workflows/integration-test.yml

```diff
@@ -18,20 +18,36 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
```

Comment:
The matrix used here for Apple platforms is copied verbatim out of https://github.com/awslabs/aws-sdk-swift/blob/main/.github/workflows/continuous-integration.yml .